panel/widgets/chat.py

```diff
@@ -273,9 +277,9 @@ def __init__(self, **params):
         self.reaction_icons.link(self, value="reactions", bidirectional=True)
         self.param.trigger("reactions")
 
-        render_kwargs = dict(
-            inplace=True, stylesheets=self._stylesheets
-        )
+        render_kwargs = {
+            "inplace": True, "stylesheets": self._stylesheets
```

Comment:
Using `{...}` is faster than `dict(...)` as you avoid a function call. In practice I don't know how much it matters.

Reply:
Doesn't matter much but I'm happy to change it.